README.md

```diff
@@ -1,19 +1,19 @@
-<p align="center">
+<h1 align="center">
```

Comment:
Better to just change the p to div...

Reply:
Yeah... the issue is the markdown linter will complain unless the file starts with a `# ` or a <h1`, I can disable the markdown linter if we care about this. https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md041---first-line-in-a-file-should-be-a-top-level-heading

Reply:
That's fine.